docker/rootless/usr/local/bin/docker-entrypoint.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+[[ -x /usr/local/bin/docker-setup.sh ]] && /usr/local/bin/docker-setup.sh
```

Comment:
Shall we cancel the startup in case the setup fails?

E.g. there is no point in starting gitea without a config file.

Reply:
On the same file shoudl we run the setup script when a custom startup command is use?

Reply:
> On the same file should we run the setup script when a custom startup command is use?

Yes. One-off commands in a docker-compose context `$ dc run --rm ...` might not work otherwise.
It is common to configure the application via environment variables only and then we would lack a config file in the brand new container.